src/en/space-station-14/chat.md

```diff
@@ -0,0 +1,637 @@
+# Chat and Other Messaging
+
+## What is Chat?
+Chat and other messages sent between players are grouped into the following types:
+
+### "In round" messages:
+
+* Local: Normal verbal communication.
+* Whisper: Quiet verbal communication that is very short range and features occlusion mechanics.
+* Emote: Visual communication via body langage. Requires line-of-sight.
+* LOOC: Out-of-character chat that is supposed to be used for asking for help or "I need to go AFK" messages, but is actually used for being salty.
+* Radio: Wireless communication via hub servers or other technology that can be listened to via equipment or innate powers. Covers all "role-specific" communications like Binary for robots or the Syndicate channel for traitors and nuclear operatives.
+    * Equipment radio covers radio that is granted by having access to a worn piece of equipment, such as a headset.
+    * Internal radio covers radio that is granted by some internal power, like Binary for robots or "nearly everything" for Nar'sie.
+    * Device radio covers radio that sends and receives from a specific location in the game, like handheld radios or intercoms.
+* Dead: The place ghosts talk where players can't hear them (and the place admins post when they want to be seen by dead people)
+* Announcements: Various announcements from automated messages through to adminbus messages or comms console notifications.
+* Background: Where automated chats like vending machine announcements are issued. Essentially identical to local chat, but these messages are "not real" and do not go into chat logs or get message IDs (see below)
+* Prayer: Messages sent from players in-game to admins via some artifact like the red phone or the maint altar.
+* Subtle: Messages targeted at one specific player, currently solely an adminbus feature.
+
+These chat types cover messages that are "round-scoped". This means they relate to something that shows up in replays and are "things that happened in that particular round".
+
+### "Out of round" messages
+
+* OOC: Lobby comms for people out-of-game. Usually switched off by cvar during games.
+* Admin: Admin notices, from all-player bwoinks to requests for feedback on events.
```

Comment:
Either I'm misreading this, or you may have confused the purpose of admin chat

Admin chat (nicked bubblegum chat due to it being pink and making a popping sound on each message sent) is meant as a chat for admins to talk to each other in game with other admins. Players/deadmined should be unable to see this chat.

Reply:
Oh, fair. Tbf I've not touched this aspect of the code at all. Will separate admin chat from bwoinks